src/main/java/Ui.java

```diff
@@ -0,0 +1,51 @@
+public class Ui {
+    private String response;
+
+    private void getResponseDirect() {
+        System.out.println("    ____________________________________________________________\n     "
```

Comment:
Nice way of abstracting the annoying borders! I've seen that you rewrite the same borders for all of your exceptions as well. Maybe you could do the same form of abstraction to them as well?